### DISCUSSION TITLE
Additional options

### DISCUSSION THREAD

**Discussion by NotValdemaras** [vdemichev/DiaNN#355] [Q&A] [answered]
Hi,

I recently started using DIA-NN and wanted to ask if there is some list of all the possible "Additional options" that are available for it? For example like the "-ms1-base-profile" that allows DIA-NN perform MS1 quantification.

Best regards,
Valdemaras

**Comment by vdemichev** [comment]
Hi Valdemaras,

Yes, please see the command line reference here https://github.com/vdemichev/DiaNN#command-line-reference
Some options are experimental and not yet listed, e.g. --ms1-base-profile can theoretically improve a bit the performance with MS1-optimised methods. But MS1 quantification works fine without it too.

Best,
Vadim

**Comment by NotValdemaras** [comment]
Hi Vadim,

Thank you for the answer, that clears it up.  But without this option the precursor intensity is based on the top 3 correlated fragment ions or did I misunderstand the method description?

Best regards,

Valdemaras

**Comment by vdemichev** [comment, marked as answer]
This option does not affect quantification directly. Yes, Precursor.Normalised is based on top 3 correlated fragments as described in the DIA-NN paper.
